### PR TITLE
11 delete comments

### PR DESCRIPTION
**Pull request by jaketysome**

### CODE REVIEW

File: src/components/CommentCard.jsx [hunk 1]
```diff
@@ -1,26 +1,50 @@
...
+            setComments((currComments) => {
+                return currComments.filter((comment) => {
+                    return comment.comment_id !== commentId;
+                })
+            })
+        }).catch((err) => {
+            setIsDeleting(false);
+            setCommentDeleted(false);
+            setDeleteError(true);
+        })
```
Comment (popatre) [inline src/components/CommentCard.jsx]: Could you either lock the delete button, or replace it with the loading thing when you click it, to stop it being clicked multiple times on a slow connection? I can spam it loads of times at the moment if the internet is slow.

File: src/App.js [hunk 2]
```diff
@@ -7,18 +7,23 @@ import ArticlesList from "./components/ArticlesList";
 import SingleArticle from "./components/SingleArticle";
 
 function App() {
-  const [user, setUser] = useState({username: "grumpy19", name: "Paul Grump", avatar_url: "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013"});
+  const [currUser, setCurrUser] = useState({
+    username: "grumpy19",
+    name: "Paul Grump",
+    avatar_url: "https://vignette.wikia.nocookie.net/mrmen/images/7/78/Mr-Grumpy-3A.PNG/revision/latest?cb=20170707233013"
+    });
+
```
Comment (popatre) [inline src/App.js]: Could you add a login/logout - even just to sign you in as this user. Can then check that you can't delete any comments if not signed in

**Review by popatre — CHANGES_REQUESTED** [review]
Some good code on the go here. Couple of things to add please:

- [x] Could you either lock the delete button, or replace it with the loading thing when you click it, to stop it being clicked multiple times on a slow connection? I can spam it loads of times at the moment if the internet is slow. 

- [x] Could you add a login/logout - even just to sign you in as this user. Can then check that you can't delete any comments if not signed in

**Review by roisinoreilly — APPROVED** [review]
this looks excellent great job, merge when ready!

**Pull request merged**